charts/pact-broker/templates/_helpers.tpl

```diff
@@ -43,17 +43,6 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- include "common.names.dependency.fullname" (dict "chartName" "postgresql" "chartValues" .Values.postgresql "context" $) -}}
 {{- end -}}
 
-{{/*
-Create the name of the service account to use
-*/}}
-{{- define "broker.serviceAccountName" -}}
```

Comment:
Why are we removing this?

Reply:
Because it isn't being used if we are setting the default just in the values.yaml per your original comment on the PR.

Reply:
I had added it to simplify the references to serviceAccount name, but following the general idea you had laid out where we wanted to keep values.yaml as the default rather than using a templating function to default it.

This does result in the problem I mentioned where the SA isn't guaranteed to be a uniquely named object because it isn't using chart.fullName or some equivalent.